router/package_request.go

```diff
@@ -59,7 +62,7 @@ func newPackageRequest(args newPackageRequestArgs) (*packageRequest, error) {
 
 	// Only go out to github to get the matched SHA if the matched SHA is
 	// necessary.
-	if isGoGetRequest(args.req) {
+	if isGoGetRequest(args.req) || isGitRequest(parts) {
```

Comment:
I thought we agree'd we were getting rid of the the `isGoGetRequest` check here? I've removed it in my PR